README.md

```diff
@@ -343,6 +343,23 @@ If the device is listed as *Unknown*, then the device has not been fully discove
 * **The device is not in the database.** If the device attributes show that this device has a valid manufacturer ID, device ID and type, then this is likely the case (eg. you see a label like "*Z-Wave node 1 (0082:6015:020D::2.0)*"). Even if the device appears to be in the database, some manufacturers use multiple sets of references for different regions or versions, and your device references may not be in the database. In either case, the database must be updated and you should raise an issue to get this addressed.
 * **The device initialisation is not complete.** Once the device is included into the network, the binding must interrogate it to find out what type of device it is. One part of this process is to get the manufacturer information required to identify the device, and until this is done, the device will remain unknown. For mains powered devices, this will normally occur quickly, however for battery devices the device must be woken up a number of  times to allow the discovery phase to complete. This must be performed with the device close to the controller and you should refer to the device manual for information on waking up the device.  
 
+## Network bandwidth, volume of messages and timeouts
+
+When configuring individual devices and writing automation scripts the following may help avoid a common pitfall of transmitting too much data in a short time.
+
+The controller stick can only be sent one message at a time from the binding to be transmitted to the network.  Any command sent when another message is being transmitted is placed in a queue in the binding.
+
+A message can take anything from just over 50 ms to many 100s of ms to be transmitted and a response to come back from the node across the network. There is therefore a finite number of messages that can be sent and received in a second while maintaining a sensible depth of queue. If too many messages are sent for too long a period then there will be noticeable pause when manually controlling devices.
+
+If a queue has many messages from an automation script, a user should not be surprised if there is a delay when interactively controlling nodes while the script runs. If the automation will run during a period when manual control may also be used, a pause between each command sent by the automation will improve user experience as it makes it less likely that the interactive command is queued behind many commands.
```

Comment:
This comment about adding the delay isn't really true. The binding will not send messages until the previous response is received, so there is no need for a delay. Additionally, the binding will prioritise messages, so adding a delay won't necessarily move things around as the user expects as order is not maintained. I would suggest to remove this part as it's likely to cause more confusion than assistance.